lib/project_types/node/commands/connect.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module Node
+  module Commands
+    class Connect < ShopifyCli::SubCommand
+      def call(*)
```

Comment:
I'm wondering a bit whether it makes sense to keep the node and rails-specific classes. It seems we're mostly adding them to show the warning and the completion message, both of which might make sense in `default_connect`. What do you think?

Reply:
Since this functionality will be used in Themes (and other project types), it makes organizational sense to split each project type into its own command and do anything project-related in there.

Reply:
I think it makes complete sense to support custom commands, but is there a difference between node / rails and the other versions of `connect`? Are there other versions at the moment?

Reply:
No, there are no other versions at the moment.

Reply:
It's better to think of it as the default being a fallback to app-like behaviour, since it is the safest and simplest choice, as opposed to Node / Rails only doing the default behaviour.